ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -482,6 +482,16 @@ FAN_SS_VSC_SCRSS:
   - VSC
   - SCRSS
 
+FAN_SFSS_SFVSC_SSPC_FDPSM:
+  description: "Fresh air fan with supply air static pressure sensor based VFD Modulation."
+  is_canonical: true
+  implements:
+  - FAN
+  - SFSS
```

Comment:
since this is on a `FAN` type, the use of `SFSS` and `SFVSC` is excessive (use `SS`).

Reply:
Changes Done